docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -517,14 +525,21 @@ The steps below are necessary for manually loading images of modules connected f
 
 1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the command `dhctl mirror-modules`.
 
-   `dhctl mirror-modules` pulls only versions of modules available in the module release channels at the time of copying.
+   `dhctl mirror-modules` pulls only versions of modules available in the module release channels at the time of copying if the `--modules-filter` flag was not set.
 
    The following command will pull module images from the source described in the `ModuleSource` resource located in the `$HOME/module_source.yml` file:
 
    ```shell
    dhctl mirror-modules -d /tmp/d8-modules -m /tmp/module_source.yml
    ```
 
+   To download only a set of specific modules of specific versions, use the `--modules-filter` flag, passing the set of required modules and their versions, separated by the `;` character.
+   For example:
+
+   ```shell
+   dhctl mirror-modules -d /tmp/d8-modules -m /tmp/module_source.yml --modules-filter="deckhouse-admin:v1.0.0;deckhouse-admin:v1.3.3; sds-drbd:v0.0.1"
```

Comment:
--modules-filter="deckhouse-admin:v1.0.0`;`deckhouse-admin:v1.3.3`; `sds-drbd:v0.0.1"

Is there a space needed between versions? In the example above, it is there in one case and missing in the other. Or does it make no difference? (This applies to the Russian version as well.)

Reply:
This space is left there intentionally, to demonstrate that it is possible to use spaces between expressions